mobile/lib/common/loading_screen.dart

```diff
@@ -38,25 +39,27 @@ class _LoadingScreenState extends State<LoadingScreen> {
       if (isSeedFilePresent) {
         runBackend(context).then((value) {
           logger.i("Backend started");
-        });
+
+          if (!hasEmailAddress) {
```

Comment:
Given the size of the PR, I think these changes should be in its own PR.

Reply:
Hmm.. you mean the full backup could have been separated into a separate PR?